packages/components/BaseButton.tsx

```diff
@@ -10,27 +10,27 @@ interface Props {
 
 const VARIANT_MAP: Record<
   Variant,
-  { padding: number; color: string; fontColor: string }
+  { paddingY: number; color: string; fontColor: string }
 > = {
-  confirm: { padding: 6, color: '#46F70C', fontColor: 'white' },
-  correct: { padding: 4, color: '#FFD600', fontColor: 'white' },
-  blank: { padding: 4, color: '#ffffff', fontColor: 'black' },
-  number: { padding: 4, color: '#000000', fontColor: 'white' },
+  confirm: { paddingY: 8, color: '#46F70C', fontColor: 'white' },
```

Comment:
temos cores disponiveis [aqui](https://www.figma.com/file/Y65seH9C3oY9Bcbxm97N8S?embed_host=notion&kind=&node-id=823%3A9205&viewer=1)
```suggestion
  confirm: { paddingY: 8, color: '#38a169', fontColor: 'white' },
```

Reply:
good to know! I will change that